data/sql/updates/pending_db_world/rev_1587846457474224200.sql

```diff
@@ -1,8 +1,6 @@
 INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1587846457474224200');
 
 UPDATE `gameobject_template` SET `Data0`=1505 WHERE `entry`=188112;
-DELETE FROM `gameobject_template` WHERE `entry`=300184;
```

Comment:
Why not remove these?
Do you play any part in the quest?

Reply:
@Kitzunu Has been removed there is no such ID

Reply:
![image](https://user-images.githubusercontent.com/24550914/80292191-ea87ea80-8754-11ea-84c4-81fffef2e106.png)

The error you got was because those that you removed had an entry in gameobject_template_addon that did not get removed

Reply:
@Kitzunu Correct now that I realize I'll update Thanks!

Reply:
👏